packages/vaadin-details/test/details.test.js

```diff
@@ -129,6 +133,28 @@ describe('vaadin-details', () => {
       details.opened = true;
       expect(content.getAttribute('aria-hidden')).to.equal('false');
     });
+
+    it('should set unique id on the content', () => {
```

Comment:
Please create another `describe` suite for this test, as it uses different fixture. Something like

```js
describe('unique IDs', () => {
  const idRegex = /^vaadin-details-content-\d+$/
  let container, details;
  
  beforeEach(() => {
    container = fixtureSync(`
      <div>
        <vaadin-details>
          <div slot="summary">Summary</div>
          <input>
        </vaadin-details>
        <vaadin-details>
          <div slot="summary">Summary</div>
          <input>
        </vaadin-details>
      </div>
    `);
    details = container.querySelectorAll('vaadin-details');
  });
});
```

Reply:
Ah ok will do 👍

Reply:
done